projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -84,11 +84,14 @@ export async function jobRequest(endPoint: string): Promise<void> {
           data['message'],
           [Dialog.okButton()]
         );
+        return '';
       }
     } catch (error) {
       showError(error);
+      return '';
     }
   }
+  return '';
```

Comment:
nit: Can we refactor this part a bit in order to not return empty string in 3 places? I think the last one should do the work for the upper two as well.